front/app/components/FormBuilder/components/FormBuilderToolbox/index.tsx

```diff
@@ -266,16 +266,15 @@ const FormBuilderToolbox = ({
           inputType="rating"
           disabled={isCustomFieldsDisabled}
         />
-        {/* TODO: Re-enable once the Sentiment Question is fully released */}
-        {/* <ToolboxItem
```

Comment:
Moved the commenting out to the FormBuilderConfig's themselves, so we can enable this for the community monitor with this PR.